test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.cs

```diff
@@ -99,13 +100,70 @@ public class HubConnectionTests
         }
 
         [Fact]
+        public async Task CancelStartAsync()
```

Comment:
These tests are a good start but we need some test that cancel a task after successfully starting the operation. 

Also we should ensure that the connection is in the expected stated after cancellation.

Reply:
This still has been addressed - I see new tests but they all pass an already cancelled CancellationToken. In real life often times the user will be using a CancellationTokenSource and call methods with non-cancelled token that might be canceled after the method was called later.